monitor/grafana/zeebe.json

```diff
@@ -1428,802 +1355,791 @@
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": "0",
+          "show": true
+        },
+        {
+          "$$hashKey": "object:2346",
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "description": "Shows how many positions are not updated by exporters per partition. Means what is the difference between last committed exporter position and committed log position.",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {},
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 1000
               }
-            },
-            "overrides": []
-          },
-          "gridPos": {
-            "h": 4,
-            "w": 12,
-            "x": 0,
-            "y": 9
-          },
-          "id": 192,
-          "options": {
-            "orientation": "auto",
-            "reduceOptions": {
-              "calcs": [
-                "last"
-              ],
-              "fields": "",
-              "values": false
-            },
-            "showThresholdLabels": false,
-            "showThresholdMarkers": true,
-            "text": {}
-          },
-          "pluginVersion": "7.4.5",
-          "targets": [
-            {
-              "expr": "zeebe_exporter_last_exported_position{namespace=~\"$namespace\", partition=~\"$partition\"} - zeebe_exporter_last_updated_exported_position{namespace=~\"$namespace\", partition=~\"$partition\"}",
-              "format": "time_series",
-              "instant": true,
-              "interval": "",
-              "intervalFactor": 1,
-              "legendFormat": "{{exporter}} {{partition}}",
-              "refId": "A"
-            }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 4,
+        "w": 12,
+        "x": 0,
+        "y": 9
+      },
+      "id": 192,
+      "options": {
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "last"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "showThresholdLabels": false,
+        "showThresholdMarkers": true,
+        "text": {}
+      },
+      "pluginVersion": "7.4.5",
+      "targets": [
+        {
+          "expr": "zeebe_exporter_last_exported_position{namespace=~\"$namespace\", partition=~\"$partition\"} - zeebe_exporter_last_updated_exported_position{namespace=~\"$namespace\", partition=~\"$partition\"}",
```

Comment:
Here also we can use `max by (partition)`. We don't have to show these values for the followers.

Reply:
Would using max guarantee we're only showing the leader? Isn't there a case here where on failover you would still show the old leader?

Reply:
You are right. If the new leader recovers from an old snapshot, it is possible that its exporter positions are behind. But showing this metrics for followers is also confusing, because zeebe_exporter_last_updated_exported_position is never updated on follower.

Reply:
Is there a way to find the current leader, use the result in this query?

Reply:
Since it's not a label, I wouldn't know how. We can try pulling it from a different metric I guess? :smile:

Reply:
For now we can chose from the above two options - using max or show values for the followers. You can chose which one is less evil :smile:

Reply:
I think it's less evil to show follower values, since it's simply less confusing if we clearly identify the pod they come from. If we aggregate, I think you might get confusing values and it won't be clear that it's because of how the values are aggregated.

Reply:
So we could `max by (pod, exporter, partitionId)`, wdyt? And add `{{pod}}` in the label.

Reply:
:+1:

Reply:
So does that mean you approved? :smile: